src/views/new-evaluation/newEvaluation.js

```diff
@@ -96,29 +100,29 @@ const selectConstructor = function(option= {}){
 }
 
 
-const dropDownConstructor = function (option={}, selectContent = {}){
+const DropDownConstructor = function (option={}, selectContent = {}){
     const result = []
     for (var i = 0; i < option.length; i++){
         result.push(`
         <li class="legend-box-drop">
             <label for="${option[i].selectName}"> ${option[i].labelTitle}</label>
             <select name="${option[i].selectName}">
-            ${selectConstructor(selectContent.Values)}
+            ${SelectConstructor(selectContent.Values)}
             </select>
         </li>
         `)
     }
     return result.join('')
 }
 
-const newEvaluationForm = function (option = {}){
+const NewEvaluationForm = function (option = {}){
     var result = []
     for(var i = 0; i < option.length; i++){
```

Comment:
Unfortunately I have been outside almost all night long and didn't managed to change the code asap as I promised during this day of work :).
Hopefully the new version I got ATM is good enough 👍 used everywhere I could maps and arrow keys to make them as short and as readable as possible